### PR TITLE
Draft: Notes for 15.2 - Double Integrals over General Regions

### PR DESCRIPTION
**Pull request by boomanaiden154**
Currently a work in progress. Haven't finished all the sections that I want to get to at the moment. Should be able to wrap everything up tomorrow morning though. My wording is probably a little (a lot?) more verbose than it needs to be here. Need to probably add in some more/better diagrams too.

### CODE REVIEW

File: Math/Calculus/Notebooks/double_integration_general_regions.ipynb [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
...LQDQahwyAYAmdNppyZZblveOPggDAAAAAAAAAAAAAAAAAAAAAAAAAAD0Vv/v/yXf+155P//8pF+/ej0A0GocMgGAJrTFFslZZ5X3W29NfvObej0AAAAAAAAAAAAAAAAAAAAAAAAAAACtYPbs8rbNNskpp9RrAYBW5JAJADSpc89NBgxY97Z2bXLZZXV7AAAAAAAAAAAAAAAAAAAAAAAAAAAAmtmjjya33Vbezz472Wyzej0A0IocMgGAJrXTTslJJ5X3665LnnuuXg8AAAAAAAAAAAAAAAAAAAAAAAAAAEAzu+yypK1t3dvAgcnQoXV7AKAVOWQCAE3sggvK2yuvJFdcUa8FAAAAAAAAAAAAAAAAAAAAAAAAAACgWT37bLJoUXn/wheS7bevlgMALcshEwBoYvvumxx1VHm//PLkz3+u1wMAAAAAAAAAAAAAAAAAAAAAAAAAANCM5s9PVqxY99ZoJBdcULcHAFqVQyYA0ORGjixvzz+ffPGL9VoAAAAAAAAAAAAAAAAAAAAAAAAAAACazQsvJAsXlvdPfjLZZ59qOQDQ0hwyAYAm9773JQcdVN7nzClf+gQAAAAAAAAAAAAAAAAAAAAAAAAAAOjprrwyWb68vI8cWa8FAFqdQyYA0OQajWTMmPK+ZEnyla/U6wEAAAAAAAAAAAAAAAAAAAAAAAAAAGgWL7+czJ1b3g89NHnve+v1AECrc8gEAFrAxz6WvP3t5X3mzGT16no9AAAAAAAAAAAAAAAAAAAAAAAAAAAAzeDaa5PnnivvY8fWawGAnsAhEwBoAX36JGPGlPfHHktuuaVeDwAAAAAAAAAAAAAAAAAAAAAAAAAAQHdbuTKZPbu8v+tdyeGH1+sBgJ7AIRMAaBHHHZfstVd5nzEjWbu2Xg8AAAAAAAAAAAAAAAAAAAAAAAAAAEB3uvHG5Mkny/vYsUmjUa8HAHoCh0wAoEX065eMHl3eH3ooufPOej0AAAAAAAAAAAAAAAAAAAAAAAAAAADdZc2a5JJLyvvb3pYcdVS9HgDoKRwyAYAWctJJyRveUN6nTUva2ur1AAAAAAAAAAAAAAAAAAAAAAAAAAAAdIdbb00WLy7vY8YkfXwTOwBsML8+AaCFDByYjBhR3n/0o+Tee+v1AAAAAAAAAAAAAAAAAAAAAAAAAAAA1NbWlkyfXt733DM5/vh6PQDQkzhk
```
Comment (justincl2003) [inline Math/Calculus/Notebooks/double_integration_general_regions.ipynb:70]: accuracy

**Comment by justincl2003** [comment]
Looks great overall. Maybe show the "this is the riemann sum" "now we can take the limit of it" in latex and less paragraph form, and a little less verbose formalness in the wording like you said. Hard to tell what it looks like without seeing everything rendered though. 

Also, are the limits of each integral the "boundary curves" of the regions in terms of the opposite variable? if so, could say its just a step up from rectangles because rectangles do technically use equations for lines as their limits in the integrals.

**Comment by boomanaiden154** [comment]
> Looks great overall. Maybe show the "this is the riemann sum" "now we can take the limit of it" in latex and less paragraph form, and a little less verbose formalness in the wording like you said. 

Definitely a good point. Using more mathematical notation does increase the density of stuff and also improves the readability from what I've seen.

> Hard to tell what it looks like without seeing everything rendered though.

You should be able to pull the branch on your end and render it locally so that you can see the changes and then just comment on the relevant lines. That's what I've been doing. Ping me on Discord if you want me to walk you through the process.

> Also, are the limits of each integral the "boundary curves" of the regions in terms of the opposite variable? if so, could say its just a step up from rectangles because rectangles do technically use equations for lines as their limits in the integrals.

Only the limits of integration for one of the integrals is the boundary curve. The outer integral has constant limits of integration. You could kind of say it's just a step up from rectangles, but given that the equations for lines there are just constants, it's a little bit more of a leap in my opinion.

**Comment by justincl2003** [comment]
> Only the limits of integration for one of the integrals is the boundary curve. The outer integral has constant limits of integration. You could kind of say it's just a step up from rectangles, but given that the equations for lines there are just constants, it's a little bit more of a leap in my opinion.
Alright, leave that out. probably just confuses things a bit

> You should be able to pull the branch on your end and render it locally so that you can see the changes and then just comment on the relevant lines. That's what I've been doing. Ping me on Discord if you want me to walk you through the process.

Ohhh, ok. Np, i can figure it out. Cool.

**Comment by justincl2003** [comment]
> Only the limits of integration for one of the integrals is the boundary curve. The outer integral has constant limits of integration. You could kind of say it's just a step up from rectangles, but given that the equations for lines there are just constants, it's a little bit more of a leap in my opinion.

Also i looked into it a bit more and i think it fits as an intuition, because one of the integrals is exactly accounting for integrating between the curves and the other between the limits in the opposite direction. You could include an example region D shown in 2d thats defined by 2 curves and 2 constant limits instead of the circle maybe. But up to you, I think the main thing is really just less paragraphs (and even this is only really the one under the circle diagram)

**Comment by boomanaiden154** [comment]
> Alright, leave that out. probably just confuses things a bit

I think I edited this one out. It was the statement in the last sentence of the introduction, right? Do you mind confirming? 

> You could include an example region D shown in 2d thats defined by 2 curves and 2 constant limits instead of the circle maybe. But up to you, I think the main thing is really just less paragraphs (and even this is only really the one under the circle diagram)

I've kept the circle for now because I don't want to get into any of the details at that point and focus purely on the partition problem. I've added more diagrams towards the end, including a labelled one that shows the partitions in a worked example.

**Review by justincl2003 — APPROVED** [review]
> "understand what the norm of the partition ||P|| is"

use single $ instead of double so that it stays in line and as part of the text body i think that looks better here

for property 5, with R1 and R2, clarify that they must be nonoverlapping

right above the properties section, missing a ']' close bracket after [h1,h2

looks great, add to website!

**Pull request merged**